force-app/main/default/lwc/accountModelSettings/accountModelSettings.js

```diff
@@ -1,20 +1,87 @@
-import { LightningElement, api, track } from "lwc";
+import { LightningElement, api, track, wire } from "lwc";
+
+import getAccountModelSettingsViewModel from "@salesforce/apex/AccountModelSettingsController.getAccountModelSettingsViewModel";
+
 import stgAccountModelSettingsTitle from "@salesforce/label/c.stgAccountModelSettingsTitle";
+import stgAccModelTitle from "@salesforce/label/c.stgAccModelTitle";
+import stgHelpAccountModel from "@salesforce/label/c.stgHelpAccountModel";
+import stgAdminAccountRecordType from "@salesforce/label/c.stgAdminAccountRecordType";
+import stgHelpAdminRecType from "@salesforce/label/c.stgHelpAdminRecType";
+import stgAccountRecordTypeSupportsHHAddress from "@salesforce/label/c.stgAccountRecordTypeSupportsHHAddress";
+import stgHelpHouseholdRecType from "@salesforce/label/c.stgHelpHouseholdRecType";
+import stgOptSelect from "@salesforce/label/c.stgOptSelect";
 
 export default class AccountModelSettings extends LightningElement {
     isEditMode = false;
     affordancesDisabledToggle = false;
 
+    @track accountModelSettingsViewModel;
+
+    labelReference = {
+        accountModelSettingsTitle: stgAccountModelSettingsTitle,
+        defaultAccountModelTitle: stgAccModelTitle,
+        defaultAccountModelDescription: stgHelpAccountModel,
+        adminAccountModelTitle: stgAdminAccountRecordType,
+        adminAccountModelDescription: stgHelpAdminRecType,
+        hhAccountModelTitle: stgAccountRecordTypeSupportsHHAddress,
+        hhAccountModelDescription: stgHelpHouseholdRecType,
+        comboboxPlaceholderText: stgOptSelect,
+    };
+
+    inputAttributeReference = {
+        defaultAccountModelComboboxId: "defaultAccountModel",
+        adminAccountModelComboboxId: "adminAccountModel",
+        hhAccountModelComboboxId: "hhAccountModel",
+    };
+
     get affordancesDisabled() {
         if (!this.isEditMode || this.affordancesDisabledToggle === true) {
             return true;
         }
         return undefined;
     }
 
-    labelReference = {
-        stgAccountModelSettingsTitle,
-    };
+    @wire(getAccountModelSettingsViewModel)
+    accountModelSettingsViewModel({ error, data }) {
+        if (data) {
+            this.accountModelSettingsViewModel = data;
+        } else if (error) {
+            //console.log("error retrieving accountmodelsettingsvmodel");
```

Comment:
Saw this while working on patterns. Is this on purpose? @m-kinkade

Reply:
@xinnanlisf yes, for now.  I'm not sure how we want to handle an error in this scenario so it will be addressed in a future iteration.